style.css

```diff
@@ -16,33 +22,75 @@
 }
 
 .cocogoose {
-  font-family: 'Cocogoosetrial', sans-serif;
+  padding-top: 8%;
+}
+
+.offcanvas-title {
+  margin-right: 30%;
 }
 
 .navbar {
-  background-color: #d3d3d3;
+  background-color: #f8f9fa;
+}
+
+.logo-img {
+  width: 100%;
+  height: 100%;
+}
+
+.logo-img3 {
+  width: 100%;
+  height: 100%;
 }
 
 .background-intro {
   background: url(./images/background-mobile2.png) no-repeat;
-  background-size: 100%;
+  background-position: center;
+  background-size: cover;
+  height: 100vh;
+}
```

Comment:
- [Optional ] Well done implementing most of the requirements however the text in the About section for the mobile version is still not well positioned. To fix this issue as requested by the previous reviewer I recommend deleting `height: 100vh` from the `background-intro` class.

![review](https://user-images.githubusercontent.com/53347966/233512681-e47d03ee-9f20-4649-8724-c9666a5478cc.JPG)

Reply:
Did it!!!, thank you!!!